ci/unit_tests.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash -ie
+
+# Since we are using the system jruby, we need to make sure our jvm process
+# uses at least 1g of memory, If we don't do this we can get OOM issues when
+# installing gems. See https://github.com/elastic/logstash/issues/5179
+export JRUBY_OPTS="-J-Xmx1g"
```

Comment:
@jakelandis for "at least" you'd want `Xms` here :)

Reply:
In this case, I think we should let the memory grow naturally. This memory setting is only used by bundler and don't want to consume memory unless we need to.  I don't know how much memory is actually needed. (this has been like this for a while)

Reply:
@jakelandis I have to problem with the implementation, just the wording of the comment was weird :D Only setting `Xmx` here is fine with me :)